### PR TITLE
🏇 Personal Profile: Styling the webpage

### LINKED ISSUE #137: 🏇 Personal Profile: Styling the webpage
The personal profile page needs to be styled better.

### PR DESCRIPTION
**Pull request by ShirleyFichman**
In this PR I updated the UI for the personal profile page:

- Updated the detail view 
- Added a `.css` file
- Included the `.css` file in the base template

image

Depending on #132
Close #137

### CODE REVIEW

**Comment by paOmer** [comment]
Hi @ShirleyFichman, The webpage looks very good!
But since you've linked the css file to the base template, it effects the entire website and makes unwanted changes to some of the webpages.
Here are 2 options that I see in order to deal with it:

- Link the css file directly to the personal profile page, this would mean to add another content block to the `<head>` tag of the base template and assign your css through there.
- The other option will be to make all the css changes in the main css file and to make sure they wont damage any other webpage view.

Also I recommend to use less of direct CSS changes and apply more bootstrap classes.

**Review by paOmer — CHANGES_REQUESTED** [review]
@ShirleyFichman The webpage looks awesome! nice work.
please add a test that checks for an invalid profile ID url

**Comment by paOmer** [comment]
> @ShirleyFichman The webpage looks awesome! nice work. please add a test that checks for an invalid profile ID url. Look here for an example https://github.com/redhat-beyond/JobSeeker/blob/main/feed/tests/test_feed_views.py#L51-L58

Just noticed that the profile detail view PR is still open so I guess this changes are more relevant to that, I'll ask for tehm there.

**Review by paOmer — APPROVED** [review]
Looks good!

**Review by paOmer — APPROVED** [review]

**Review by DeanBiton — APPROVED** [review]

File: personal_profile/tests.py [hunk 1]
```diff
@@ -68,3 +64,17 @@ def test_profile_is_deleted_when_user_is_deleted(self, profile_1, user_1):
...
         test_user.delete()
         assert test_profile not in PersonalProfile.objects.all()
+
+
+@pytest.mark.django_db
+class TestProfileDetailView:
+    def test_detail_view_page_entrypoint(self, profile_1, user_1, client):
+        # Testing to see if a valid user id gets a valid detail view page
+        response = client.get(PROFILE_DETAIL_URL + str(profile_1.id) + '/')
+        assert response.status_code == 200
```
Comment (Yarboa) [inline personal_profile/tests.py:74]: Is there a way to verify valid user gets valid detail view from  response? 
Maybe assert returned template? 
personalprofile_detail.html

You could also verify the returned object from DetailedView,
you should be able to read this in response
personalprofile

**Review by Yarboa — CHANGES_REQUESTED** [review]
Nice Work, take more care with client tests

File: personal_profile/tests.py (hunk 1 above)
Comment (ShirleyFichman) [reply personal_profile/tests.py:77]: Added the test of the template.
Regarding the 2nd suggestion- tried but couldn't understand how to access the DetaileView object from the response.

**Comment by Yarboa** [comment]
Please rebase

File: personal_profile/tests.py (hunk 1 above)
Comment (Yarboa) [reply personal_profile/tests.py:77]: Please look at template
It is reading the following data
personalprofile.user.username and other files, please review UI talk, and try to Can you receive the personalprofile from response.context?

**Review by Yarboa — DISMISSED** [review]
LGTM

**Comment by Yarboa** [comment]
> LGTM

Sorry, but seems there is one failing test

**Comment by ShirleyFichman** [comment]
> > LGTM
> 
> Sorry, but seems there is one failing test
@Yarboa I rebased and now it passed

**Pull request merged**